### DISCUSSION TITLE
Hello there, I'm doing a new view for this project.

### DISCUSSION THREAD

**Discussion by juanca2918** [InvoiceShelf/InvoiceShelf#254] [Q&A]
Hello, I just need to know how to set up a new view with its backend in the project. I have made two views to handle the backend lifecycle requests for a form in create.vue and an index called index.vue in the crud_test folder. The idea is to generate a simple CRUD to learn how the project works. In the end, I'll make a view with its backend to simulate financial services.
[generateviews.zip](https://github.com/user-attachments/files/18127485/generateviews.zip)

**Comment by juanca2918** [comment]
Initially, I started developing a simple CRUD for this repo. I just need to know how to render the view that I have been developing for the CRUD. I know there are policies bound to the view, a controller, a model, and a dynamic menu rendered by the Lavary package. However, I still don't know how to bring the new menu for the CRUD to the view in the browser. I also know that there is a GenerationMenuTrait that interacts with the dynamic menu, but as I mentioned, I don't know how the render works with the view because this repo does not use Inertia as a framework to handle the request and response.

**Reply by rihards-simanovics** [reply]
Hey @mchev, you've made some PRs with Vue changes before; could you help @juanca2918? I'm sorry, but I'm doing assessments right now and have very little time for any OSS coms.

**Reply by mchev** [reply]
Hi, I feel that it is more a "how to" than an issue.
Feel free to contact me on discord I'll see if I can do something to help

**Reply by juanca2918** [reply]
Thanks, I’ll contact you immediately.
